clients/python-wrapper/tests/integration/conftest.py

```diff
@@ -37,6 +37,14 @@ def setup_repo(storage_namespace, test_name, default_branch="main"):
     return _setup_repo(storage_namespace, test_name, default_branch)
 
 
+@pytest.fixture(scope="session")
+def setup_branch():
```

Comment:
You can just call it "test_branch" and also you can use it as is - no need to reassign it to another variable in the test